pkg/sql/planner.go

```diff
@@ -235,6 +235,12 @@ func makeInternalPlanner(
 }
 
 func finishInternalPlanner(p *planner) {
+	// If we're panicking, don't bother stopping the MemoryMonitors. We're likely
```

Comment:
this is "not bothering" to do a bunch of things, not just stopping the memory monitors.